apis/crd/v1alpha1/virtualmachine_types.go

```diff
@@ -86,6 +86,8 @@ type VirtualMachineStatus struct {
 	NetworkInterfaces []NetworkInterface `json:"networkInterfaces,omitempty"`
 	// State indicates current state of the VirtualMachine.
 	State VMState `json:"state,omitempty"`
+	// Region indicates the cloud region of the VirtualMachine.
```

Comment:
Can you update the doc and also the tabular output of VM

Reply:
Add +kubebuilder:printcolumn:name="Region",type=string,JSONPath=`.status.region` after Cloud-Provider so that VM output includes region and update getting started doc.

Reply:
Done. Also `Agented` field is missing from doc get vm output, added that too.